HelpSource/Guides/UsingQuarks.schelp

```diff
@@ -5,7 +5,28 @@ related:: Classes/Quarks, Classes/Quark, Classes/Git
 
 Quarks are packages of SuperCollider code containing classes, extension methods, documentation and server UGen plugins. The Quarks class manages downloading these packages and installing or uninstalling them.
 
-section:: Installing quarks
+warning::
+Because individual users or groups develop their own Quark for different purposes, installing multiple packages can cause problems because they conflict with each other. We recommend that you select and install only the packages that you need. It is also recommended that you install one Quark package at a time to make it easier to troubleshoot problems as they arise.
+::
+
+note::
+If you have trouble finding the Quarks package you need using the Quarks installation methods in this document, you may find the Quarks package information in the following encyclopaedia useful:
+link::https://baryon.supercollider.online::
+::
+
+section::Preliminary stages
+
+To use Quark, you shoud install git software from link::https://git-scm.com::.
+list::
+##Windows users can download and install the appropriate installation file for their environment from the above web page.
```

Comment:
... from link::https://git-scm.com/download/win::.

(Readers shouldn't have to navigate through two clicks in order to find the actual installers.)

Reply:
thanks, done!